### PR TITLE
Fix redirected links and a 404 one

### PR DESCRIPTION
**Pull request by XhmikosR**
Let me know if there are any links you want me to revert. :)

### CODE REVIEW

File: docs/guides/upstream-dns-providers.md [hunk 1]
```diff
@@ -84,5 +84,4 @@ If you care about net independence and privacy I'll suggest you give a look at :
 
 
 ### More information.
-There are even more public DNS server, you can find many (with some extra information) the [About.com website](http://pcsupport.about.com/od/tipstricks/a/free-public-dns-servers.htm). A benchmark of these DNS servers (by Gibson Research Center) is available [here](https://www.grc.com/dns/Benchmark.htm)
-There are even more public DNS server, you can find many (with some extra information) the [About.com website](http://pcsupport.about.com/od/tipstricks/a/free-public-dns-servers.htm). A benchmark of these DNS servers (by Gibson Research Center) is available [here](https://www.grc.com/dns/Benchmark.htm)
+There are even more public DNS server, you can find many (with some extra information) the [lifewire.com website](https://www.lifewire.com/free-and-public-dns-servers-2626062). A benchmark of these DNS servers (by Gibson Research Center) is available [here](https://www.grc.com/dns/Benchmark.htm)
```
Comment (PromoFaux) [inline docs/guides/upstream-dns-providers.md]:
```suggestion
There are even more public DNS server, you can find many (with some extra information) on the [lifewire.com website](https://www.lifewire.com/free-and-public-dns-servers-2626062). A benchmark of these DNS servers (by Gibson Research Center) is available [here](https://www.grc.com/dns/Benchmark.htm)
```

File: docs/main/origins.md [hunk 2]
```diff
@@ -9,10 +9,10 @@ last_updated: Sun Jan 13 18:35:14 2019 UTC
...
 
 - [`dnsmasq`](http://www.thekelleys.org.uk/dnsmasq/doc.html) - a lightweight DNS and DHCP server
-- [`curl`](https://curl.haxx.se) - A command line tool for transferring data with URL syntax
-- [`lighttpd`](https://www.lighttpd.net) - webserver designed and optimized for high performance
-- [`php`](https://secure.php.net) - a popular general-purpose web scripting language
-- [AdminLTE Dashboard](https://github.com/almasaeed2010/AdminLTE) - premium admin control panel based on Bootstrap 3.x
+- [`curl`](https://curl.haxx.se/) - A command line tool for transferring data with URL syntax
+- [`lighttpd`](https://www.lighttpd.net/) - webserver designed and optimized for high performance
+- [`php`](https://secure.php.net/) - a popular general-purpose web scripting language
+- [AdminLTE Dashboard](https://github.com/ColorlibHQ/AdminLTE) - premium admin control panel based on Bootstrap 3.x
```
Comment (PromoFaux) [inline docs/main/origins.md]: And it was with this change that I realised AdminLTE belonged to someone else now. Maybe. https://github.com/ColorlibHQ/AdminLTE/issues/2081

Comment (XhmikosR) [reply docs/main/origins.md]: Yup, so feel free to push any removals if you notice something you don't like. I noticed quite a few redirects and fixed them here.

Comment (PromoFaux) [reply docs/main/origins.md]: All good, happy to approve this one

**Pull request merged**